webapp/components/signup_user_complete.jsx

```diff
@@ -295,22 +295,6 @@ export default class SignupUserComplete extends React.Component {
         }
 
         const providedPassword = ReactDOM.findDOMNode(this.refs.password).value;
-        if (!providedPassword || providedPassword.length < Constants.MIN_PASSWORD_LENGTH) {
```

Comment:
I tried that, however the user needs to be logged in to access the config in `admin_store.jsx`, therefore I decided to leave it as server side only.